### ISSUE TITLE
filter teller-specific output by default

### ISSUE THREAD

**Issue by cnygardtw** [tellerops/teller#119]
## Feature Request

The logging of teller-specific output interferes when you are trying to capture the output if the command being run.  Example, I can't pass the output to `jq` because of the teller output:

```
% teller run -- command.sh
-*- teller: loaded variables for dev-docs using .teller.yml -*-
{ "result": "uuid" }

% teller run -- command.sh | jq -c .result
<jq parsing error because of teller debug output>
```

I'd like to be able to control the output of this logging via the `--log-level` parameter.  Logging of this sort could be info level so setting to warn or error would prevent the output.

An alternative would be to log to stderr so that I can choose the stream to process.

Kludge solution, pipe output to `grep -v "\-\*-"`, but I'd rather have the output off by default.

**Comment by kaplanelad** [comment]
Hey @cnygardtw, 
you can use `json` instead of `show` subcommand for this usecase. 
the command `teller json | jq .` should work like you describe.

**Comment by cnygardtw** [comment]
That formats the secrets.  I'm interested in capturing the output of a command like `teller run -- command.sh | jq .`

**Comment by kevbook** [comment]
You would use jq filter. Eg. `| with_entries( select(.value | has("stop_time_utc") ) )`

**Comment by kaplanelad** [comment]
@cnygardtw, the fix just released in version v1.5.3
Let me know if it works for you

**Comment by cnygardtw** [comment]
Works for me, thanks friends!
